src/version.rs

```diff
@@ -300,16 +300,17 @@ pub fn dependencies(req: &mut Request) -> CargoResult<Response> {
 
 /// Handles the `GET /crates/:crate_id/:version/downloads` route.
 pub fn downloads(req: &mut Request) -> CargoResult<Response> {
+    let (offset, limit) = req.pagination(90, 100)?;
```

Comment:
That would make more seance. What is a good format for the date as url query fragment? And, how do we parce it into a type for the sql query?